pkg/auth/authorizer/abac/abac.go

```diff
@@ -215,13 +215,13 @@ func resourceMatches(p api.Policy, a authorizer.Attributes) bool {
 }
 
 // Authorizer implements authorizer.Authorize
-func (pl policyList) Authorize(a authorizer.Attributes) error {
+func (pl policyList) Authorize(a authorizer.Attributes) (bool, error) {
 	for _, p := range pl {
 		if matches(*p, a) {
-			return nil
+			return true, nil
 		}
 	}
-	return errors.New("No policy matched.")
+	return false, errors.New("No policy matched.")
```

Comment:
this is actually a good example of the `reason` field @deads2k referred to... the "No policy matched" information is useful, but not an error